cylc/flow/task_events_mgr.py

```diff
@@ -113,8 +113,8 @@ class EventData(Enum):
     """Template variables which are available to event handlers."""
 
     Event = 'event'
-    Suite = 'suite'
-    SuiteUUID = 'suite_uuid'
+    Workflow = 'workflow'
+    WorkflowUUID = 'workflow_uuid'
```

Comment:
This is a user-facing interface.

Should put `Suite` and `SuiteUUID` back in as deprecated fields so that event handlers can support Cylc7/8.

Reply:
Good call. I've also done this for workflow (as opposed to task) events.